include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort_one_wg.h

```diff
@@ -0,0 +1,210 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _ONEDPL_parallel_backend_sycl_radix_sort_one_wg_H
+#define _ONEDPL_parallel_backend_sycl_radix_sort_one_wg_H
+
+//namespace oneapi
+//{
+//namespace dpl
+//{
+//namespace __par_backend_hetero
+//{
```

Comment:
Is there a reason that this file places everything in the global namespace?

Reply:
it is OK because this file is included from the mentioned namespace.
Please see the Alexey's comment https://github.com/oneapi-src/oneDPL/pull/709#discussion_r1051598986